flyingpigeon/processes/wps_spatial_analog.py

```diff
@@ -252,7 +252,7 @@ def _handler(self, request, response):
                           )
 
         except Exception as e:
-            msg = 'Spatial analog failed: {}'.format(e)
+            msg = 'Spatial analog failed: %s' % e
```

Comment:
Why not use format ?

Reply:
For consistency with other algorithms, to use fewer characters, and because most other examples of string formatting with error codes suggest using `'%s' % e`. I find it gives the impression that this string formatting style is specifically for errors. 

Also, if it turns out that a better method uses `traceback`, it will look better when refactored to not have the variable in parentheses.

In summary, aesthetics.

Reply:
%-formatting is considered old-style, see https://pyformat.info/
I think format is preferable going forward.

Reply:
Okay. Once I hear back from @cehbrecht on #254, I'll switch to `str.format(e)` conventions from now on.

Reply:
@huard You might find this article on the newest form of string formatting ('f-strings') interesting https://realpython.com/python-f-strings/. Perhaps we'll move to this standard in a few years.